src/Uno.UI/UI/Xaml/FontFamilyHelper.cs

```diff
@@ -1,58 +1,58 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using Windows.UI.Xaml.Media;
 
-namespace Windows.UI.Xaml
+namespace Windows.UI.Xaml;
+
+public partial class FontFamilyHelper
 {
-    public class FontFamilyHelper
-    {
 #if NETFX_CORE
-		public static Windows.UI.Xaml.Media.FontFamily Create(string familyName)
-		{
-			return new Windows.UI.Xaml.Media.FontFamily(familyName);
-		}
+	public static Windows.UI.Xaml.Media.FontFamily Create(string familyName)
+	{
+		return new Windows.UI.Xaml.Media.FontFamily(familyName);
+	}
 #elif XAMARIN
-		public static string Create(string familyName)
-		{
-			return familyName;
-		}
+	public static string Create(string familyName)
+	{
+		return familyName;
+	}
 #endif
 
 #if XAMARIN_IOS
-		/// <summary>
-		/// This methods removes the font files extensions, typically .otf or .ttf because in iOS
-	    /// you need to refer to a FontFamily via its name without extension and in Android you need the extension.
-		/// </summary>
-		/// <param name="familyName"></param>
-		/// <returns></returns>
-		public static string RemoveExtension(string familyName)
-		{
-			return familyName
-				.Replace(".otf", string.Empty)
-				.Replace(".ttf", string.Empty);
+	/// <summary>
+	/// This methods removes the font files extensions, typically .otf or .ttf because in iOS
+    /// you need to refer to a FontFamily via its name without extension and in Android you need the extension.
+	/// </summary>
+	/// <param name="familyName"></param>
+	/// <returns></returns>
+	public static string RemoveExtension(string familyName)
+	{
+		return familyName
+			.Replace(".otf", string.Empty)
+			.Replace(".ttf", string.Empty);
         }
 #endif
 
-		public static string RemoveUri(string familyName)
-		{
-			var slashIndex = familyName.LastIndexOf("/");
+	public static string RemoveUri(string familyName)
+	{
+		var slashIndex = familyName.LastIndexOf("/");
 
-			if (slashIndex != -1)
-			{
-				familyName = familyName.Substring(slashIndex + 1);
-			}
-			return familyName;
+		if (slashIndex != -1)
+		{
+			familyName = familyName.Substring(slashIndex + 1);
 		}
+		return familyName;
+	}
 
-	    public static string RemoveHashFamilyName(string familyName)
-	    {
-			var hashIndex = familyName.IndexOf("#");
+	public static string RemoveHashFamilyName(string familyName)
+	{
+		var hashIndex = familyName.IndexOf("#");
 
-			if (hashIndex != -1)
-			{
-				familyName = familyName.Substring(0, hashIndex);
-			}
-			return familyName;
+		if (hashIndex != -1)
+		{
+			familyName = familyName.Substring(0, hashIndex);
 		}
+		return familyName;
```

Comment:
As the class seems to need to be `public`, should we at least make those method `internal` ?

Reply:
https://github.com/unoplatform/uno/issues/8339